bears/python/PyImportSortBear.py

```diff
@@ -176,13 +209,38 @@ def run(self, filename, file,
             isort_settings['known_standard_library'] = (
                 known_standard_library_imports)
 
-        sort_imports = SortImports(file_contents=''.join(file),
-                                   **isort_settings)
-        new_file = tuple(sort_imports.output.splitlines(True))
+        if treat_seperated_imports_independently:
+            import_stmts = PyImportSortBear.seperate_imports(file)
```

Comment:
the revised sorting should be a separate function, so we dont have so much logic here in the run() method.